src/components/_global/BalTextInput/BalTextInput.vue

```diff
@@ -172,7 +172,7 @@ onMounted(() => {
 }
 
 .input {
-  @apply flex-grow bg-transparent overflow-hidden;
+  @apply flex-grow bg-transparent overflow-hidden w-full;
```

Comment:
Are you sure this doesn't have wider implications where the text input is used?

Reply:
hm, it looks like rather harmless change.  I checked both safari/chrome token select input, token inputs on swap page, inputs on pool creation page, vote input, everywhere it looks ok